tests/utils/helpers.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import logging
+import multiprocessing
+import unittest
+import psutil
+import signal
+import time
+
+from airflow.utils import helpers
+
+
+class TestHelpers(unittest.TestCase):
+    @staticmethod
+    def _ignores_sigterm(child_pid):
+        def signal_handler(signum, frame):
+            pass
+        signal.signal(signal.SIGTERM, signal_handler)
+        child_pid.value = os.getpid()
+        while True:
+            time.sleep(1)
+
+    @staticmethod
+    def _parent_of_ignores_sigterm(child_process_killed, child_pid):
+        child = multiprocessing.Process(target=TestHelpers._ignores_sigterm, args=[child_pid])
+        child.start()
+        while child_pid.value == 0:
+            time.sleep(0.1)
```

Comment:
Let's get rid of the sleep here, I want to avoid sleeping in tests where possible (busywaiting should be fine anyways until we parallelize tests).